src/applications/personalization/dashboard/components/RenderClaimsWidgetDowntimeNotification.jsx

```diff
@@ -36,3 +42,11 @@ export const RenderClaimsWidgetDowntimeNotification = (downtime, children) => {
   }
   return children;
 };
+
+RenderClaimsWidgetDowntimeNotification.propTypes = {
+  children: PropTypes.node,
```

Comment:
non blocking, but to make the children prop type a little more flexible I usually see it specified as:

```
  children: PropTypes.oneOfType([
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.node,
  ])
```